src/utils/data/getPropertyValues.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { Models } from 'lattice';
+
+import getPropertyValue from './getPropertyValue';
+
+const { FQN } = Models;
+
+export default function getPropertyValues(
+  entity :Map | Object,
+  fqns :any[],
+  defaultValue ? :string = ''
```

Comment:
It's not guaranteed that all property types will have the same data type, so using the same type as the default doesn't make much sense to me.

Reply:
ya that makes sense.